.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  parser: "vue-eslint-parser",
+  parserOptions: {
+    parser: "@typescript-eslint/parser",
+  },
+  extends: ["@nuxtjs/eslint-config-typescript", "plugin:prettier/recommended"],
```

Comment:
I think we should add `"extends": "eslint:recommended"` to the `extends` section. Read [this](https://eslint.org/docs/latest/use/getting-started#configuration:~:text=%7B-,%22extends%22%3A%20%22eslint%3Arecommended%22,-%7D)

Reply:
Added in https://github.com/Qiskit/qiskit.org/pull/2978/commits/919baa0ede9a4b58bf5215337b7ef0698558d879